experiments/synthea-breast-cancer/winter2022/Differentially-Private/quickstart.sh

```diff
@@ -30,13 +30,14 @@ help ()
    echo "Usage:"
    echo "   ./quickstart.sh [options]"
    echo "Options:"
-   echo "   -i <INGEST_PATH>     Ingest Data present at Path into Katsu"
-   echo "   -p <PORT>            Specify Port Number to expose. Defaults to 5000"
-   echo "   -n <SITES>           Number of Sites to federate. Defaults to 2"
-   echo "   -r <ROUNDS>          Number of rounds of trials to conduct. Defaults to 100."
-   echo "   -e <EXPERIMENT_PATH> Pass in the path to the experiment folder.Defaults to ./experiment in the root folder. Ensure this path is either an absolute path, or that it starts with './'"
-   echo "   -s                   Keep all of the data in one dataset - Useful only if -i specified"
-   echo "   -h                   Display this help text"
+   echo "   -i <INGEST_PATH>               Ingest Data present at Path into Katsu"
+   echo "   -p <PORT>                      Specify Port Number to expose. Defaults to 5000"
+   echo "   -n <SITES>                     Number of Sites to federate. Defaults to 2"
+   echo "   -r <ROUNDS>                    Number of rounds of trials to conduct. Defaults to 100."
+   echo "   -e <EXPERIMENT_PATH>           Pass in the path to the experiment folder. Defaults to ./experiment in the root folder. Ensure this path is either an absolute path, or that it starts with './'"
```

Comment:
The difference between the `EXPERIMENT_PATH` and `FEDERATED_EXPERIMENT_PATH` is not very clear; I don't think a new dev would understand what's happening here. Could you clarify (and maybe rename these variable names?) here and in the relevant documentation in `./README.md`?